src/Components.tsx

```diff
@@ -33,18 +33,25 @@ export const ConnectPaymentDetails = ({
   return wrapper;
 };
 
+export type CollectionOptions = {
```

Comment:
Ideally we'd be importing this type from `connect-js`, but that requires another PR to export... this code is simple enough, so let's keep as is for now. Sorry I didn't catch this in an earlier PR!

Reply:
Yeah, was thinking the same